host/onebox.go

```diff
@@ -714,6 +715,17 @@ func (c *temporalImpl) overrideHistoryDynamicConfig(client *dynamicClient) {
 	}
 }
 
+func (c *temporalImpl) RefreshNamespaceCache() {
```

Comment:
Might also be value in running these concurrently. Not required though.

Reply:
I thought of that so I timed it: the whole function finishes in < 10ms, so I decided it wasn't worth it.